doc/reference/data-compatibility-guide.md

```diff
@@ -1,31 +1,71 @@
 # QGreenland Dataset Compatibility Guide
 
-While QGreenland provides a curated base package of data on a variety of topics, the options for adding additional data are nearly unlimited. We offer different guidelines for data compatibility depending on what your goals are:
+While QGreenland provides a curated base package of data on a variety of topics,
+the options for adding additional data are nearly unlimited. We offer different
+guidelines for data compatibility depending on what your goals are:
 
-**1. Guidelines for publishing previously unpublished data so it is compatible with QGreenland**:
```

Comment:
Turned these bold section headers into proper markdown headers. No reason for it to be a numbered list.